dev-libs/hyphen/hyphen-2.8.8-r1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="ALTLinux hyphenation library"
+HOMEPAGE="http://hunspell.github.io/"
+SRC_URI="mirror://sourceforge/hunspell/${P}.tar.gz"
+
+LICENSE="GPL-2 LGPL-2.1 MPL-1.1"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~amd64-linux ~x86-linux"
+IUSE="static-libs"
+
+RDEPEND="app-text/hunspell"
+DEPEND="${RDEPEND}
+	dev-lang/perl
+"
+
+DOCS=( AUTHORS ChangeLog NEWS README{,.nonstandard,.hyphen,.compound} THANKS TODO )
+
+src_configure() {
+	econf $(use_enable static-libs static)
+}
+
+src_install() {
+	default
+
+	docinto pdf
+	dodoc doc/*.pdf
+
+	rm -f "${ED}"usr/lib*/libhyphen.la
+	rm -rf "${ED}"usr/share
```

Comment:
Why `-f`? Also append `|| die`. 

This looks a bit scary but I guess it's there for a reason.

Reply:
all your comments are on code changes that aren't mine. I'm just adding a patch to this existing ebuild.

Reply:
Well, I'm not in position to argue about this, but keep in mind you're revbumping thus creating a new ebuild. And it's your name that shows up as committer, so you should hold yourself up to higher standards ;)

But in the end it's really up to the dev who pushes it.

Reply:
Your administratering this PR I'm just a proxy maintainer for myspell-en, I expect you to take the decisive action here on how you would like me to proceed.

You comment implies you would prefer I rewrite  the entire ebuild in order to submit an  patch.

That's seems like allot of friction to put on contributors, but I will do the rewrite if you want me to

Reply:
>Your administratering this PR I'm just a proxy maintainer for myspell-en, I expect you to take the decisive action here on how you would like me to proceed.

I hand out suggestions, not demands ;)

>You comment implies you would prefer I rewrite the entire ebuild in order to submit an patch.

Not the whole ebuild, just few parts that could use updating.

>That's seems like allot of friction to put on contributors, but I will do the rewrite if you want me to

Normally, if you **patch existing ebuild** no other modifications are required. However, in **my mind** this is a "new" ebuild with the **revbump** and **resetted keywords**. When submitting new ebuilds, everything should be up-to-date (EAPI) and past faulties fixed.

When someone from @gentoo/office project takes a look, they **might** require you to update those parts, or they **might** merge it as-is. It's up to them to decide. However, if they want you to update this ebuild, merging **then** would be faster. It can also be they push your first commit and update hyphen package by themselves.

I'm sorry for causing a lot of noise, my intention was to get this merged faster, but I admit this could be interpret in two different ways. 
@a17r, can you give your input to this PR & to me please?